src/messageDescriptors/content.ts

```diff
@@ -32,6 +32,13 @@ export const mediaGallery = defineMessages({
     },
 });
 
+export const mediaGalleries = defineMessages({
+    title: {
+        id: 'mediaGalleries.title',
+        defaultMessage: 'Media galleries',
+    },
+});
```

Comment:
I believe we wanted to replace the existing translation with plural text. If so, it makes more sense to just replace the defaultMessage in the existing selector. This will also save us from changing the code in themes once the translations are updated.

Reply:
Yeah, that simplifies that part of code, will update!